test/storage/block_compactor_test.cpp

```diff
@@ -52,43 +52,37 @@ TEST(BlockCompactorTest, SimpleTest) {
   }
 
   storage::ArrowBlockMetadata &arrow_metadata = accessor.GetArrowBlockMetadata(block);
-  auto id_bitmap = std::make_shared<arrow::Buffer>(reinterpret_cast<uint8_t *>(
-                                                       accessor.ColumnNullBitmap(block, storage::col_id_t(2))), 1);
-  auto id_data = std::make_shared<arrow::Buffer>(reinterpret_cast<uint8_t *>(
-                                                     accessor.ColumnStart(block, storage::col_id_t(2))), 32);
-  auto id_array_data = arrow::ArrayData::Make(arrow::int64(),
-                                              arrow_metadata.NumRecords(),
-                                              {id_bitmap, id_data},
+  auto id_bitmap = std::make_shared<arrow::Buffer>(
```

Comment:
I'll merge in my changes to the test later today, which should give you random blocks.